### ISSUE TITLE
Jinja templating

### ISSUE THREAD

**Issue by AlbertDeFusco** [conda-incubator/conda-project#81]
let's consider the use of jinja templating in at least the `commands:` section and possibly other places.

**Comment by AlbertDeFusco** [comment]
@maximlt suggested having defined parameters in a command

https://github.com/conda-incubator/conda-project/issues/67#issuecomment-1423012008

**Comment by jlstevens** [comment]
I'm happy to help with this having originally added this feature to anaconda-project..

**Comment by AlbertDeFusco** [comment]
Yes, please. It might be a good time to consider reasonable support for multiline commands if you feel it is essential
